projects/vdk-plugins/vdk-trino/src/vdk/plugin/trino/trino_connection.py

```diff
@@ -109,61 +107,46 @@ def execute_query_with_retries(self, query):
         res = super().execute_query(query)
         return res
 
-    def add_num_rows_after_query(self, lineage_data, res):
-        if lineage_data:
-            lineage_data["output_num_rows_after"] = self._get_output_table_num_rows(
-                lineage_data
-            )
-            if "outputTable" in lineage_data:
-                if res and res[0] and res[0][0]:
-                    lineage_data["output_num_rows_updated"] = res[0][0]
-
     def _get_lineage_data(self, query, query_id):
-        if self._lineage_logger:
+
+        from vdk.plugin.trino import lineage_utils
+        import sqlparse
+
+        statement = sqlparse.parse(query)[0]
+
+        if statement.get_type() == "ALTER":
+            rename_table_names = lineage_utils.parse_rename_table_names(query)
+            if rename_table_names:
```

Comment:
let's add logs (debug) especially in the non covered branches (what if rename table_names is None) ?

Reply:
OK